docs/docs-next/docs/guides/automation.md

```diff
@@ -170,8 +105,8 @@ Use this table to help guide your decision:
 
 ## Next Steps
 
-- Learn more about [advanced scheduling patterns](link-to-advanced-scheduling)
-- Explore [complex sensor examples](link-to-sensor-examples)
-- Dive into [Declarative Automation best practices](link-to-declarative-automation)
+- Learn more about [advanced scheduling patterns]
```

Comment:
What's an 'advanced' pattern?

Reply:
i was thinking partitioned schedules, custom timezones?

Reply:
WDYT about adding context here using those examples? Ex: `Lean more about [advanced scheduling patterns], like partitioned schedules and custom timezones`

Reply:
I think I'll remove the next steps section right now with a TODO since I don't have full clarity on what sections contain what.